samples/charts/data-chart/scatter-series/Pages/DataChartScatterSeries.razor

```diff
@@ -0,0 +1,218 @@
+                        <option>Bubble</option>
+                        <option>Point</option>
+                        <option>Line</option>
+                        <option>Spline</option>
+                    </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (data1 != null && data2 != null)
+        {
+        <DataChart Height="100%" Width="100%"
+                   ChartTitle="GDP vs Population"
+                   IsHorizontalZoomEnabled="true"
+                   IsVerticalZoomEnabled="true"
+                   Legend="@LegendRef"
+                   @ref="DataChartRef">
+
+            <NumericXAxis Name="xAxis" 
+                          IsLogarithmic="true" 
+                          AbbreviateLargeNumbers="true"                        
+                          Title="Population" />
+            <NumericYAxis Name="yAxis" 
+                          Title="Total GDP ($)" 
+                          IsLogarithmic="true"
+                          AbbreviateLargeNumbers="true"/>
+
+        </DataChart>
+        }
+        </div>
+</div>
+
+@code {
+
+    private object data1;
+    private object data2;
+
+    private DataChart dataChart;
+    private DataChart DataChartRef
+    {
+        get { return dataChart; }
+        set
+        {
+            dataChart = value;
+            this.setSeries("Point");
+            SeriesType = "Point";
+            StateHasChanged();
+        }
+    }
+
+    private Legend legend;
+    private Legend LegendRef
+    {
+        get { return legend; }
+        set
+        {
+            legend = value;
+            StateHasChanged();
+        }
+    }
+    private string seriesType;
+    private string SeriesType
+    {
+        get { return seriesType; }
+        set
+        {
+            seriesType = value;
+            StateHasChanged();
+        }
+    }
+
+    protected override void OnInitialized()
+    {
+        LegendModule.Register(IgniteUIBlazor);
+        DataChartCoreModule.Register(IgniteUIBlazor);
+        DataChartScatterCoreModule.Register(IgniteUIBlazor);
+        DataChartInteractivityModule.Register(IgniteUIBlazor);
+        DataChartScatterCoreModule.Register(IgniteUIBlazor);
+        DataChartScatterModule.Register(IgniteUIBlazor);
+        NumberAbbreviatorModule.Register(IgniteUIBlazor);
+
+        LegendModule.Register(IgniteUIBlazor);
+        this.data1 = SampleScatterStats.GetCountriesWithHighIncome();
+        this.data2 = SampleScatterStats.GetCountriesWithLowIncome();
+    }
+
+    public void OnSeriesTypeChanged(ChangeEventArgs e)
+    {
+        var selectedSeries = e.Value.ToString();
+        SeriesType = selectedSeries;
+        this.setSeries(selectedSeries);
+    }
+
+    public void setSeries(string seriesType)
+    {
```

Comment:
there should be no code-behind to create series/axes and all these series/axes must be created in blazor code